### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by wangzaistone**
1、更新README部分的内容，围绕如何使用修改相关描述和介绍关键参数含义。
2、增加部分评估和实验结果放到doc/eval_llm_result.md中。

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,22 +21,30 @@
...
+DB-GPT-Hub is an experimental project utilizing LLMs (Large Language Models) to achieve Text-to-SQL parsing. The project primarily encompasses data collection, data preprocessing, model selection and building, and fine-tuning of weights. Through this series of processes, we aim to enhance Text-to-SQL capabilities while reducing the model training costs, allowing more developers to contribute to the improvement of Text-to-SQL accuracy. Our ultimate goal is to realize automated question-answering capabilities based on databases, enabling users to execute complex database queries through natural language descriptions.    
+
+So far, we have successfully integrated multiple large models and established a complete workflow, including data processing, model SFT (Supervised Fine-Tuning) training, prediction output, and evaluation. The code is readily reusable within this project.    
+
+As of October 10, 2023, by fine-tuning an open-source model of 13 billion parameters using this project, **the execution accuracy on the Spider evaluation dataset has surpassed that of GPT-4!**
```
Comment (csunny) [inline README.md:28]: give score? such as 83.6?

Comment (wangzaistone) [reply README.md:28]: > give score? such as 83.6?

ok， after make sure the train process has not data leakage，we give the latest best score.

Comment (csunny) [reply README.md:28]: great!

**Review by csunny — APPROVED** [review]
Nice job，LGTM🌹

**Pull request merged**